packages/suite-base/src/testing/builders/RosTimeBuilder.ts

```diff
@@ -0,0 +1,28 @@
+// SPDX-FileCopyrightText: Copyright (C) 2023-2024 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)<lichtblick@bmwgroup.com>
+// SPDX-License-Identifier: MPL-2.0
+// This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
As it is a new file, please let only our license information. We will keep the other licenses only if exist previously.